client/src/legacy/LeftAndMain.js

```diff
@@ -1508,7 +1508,13 @@ $.entwine('ss', function($) {
   });
 });
 
+// TODO: move this to an external util
+function decodeEntities(encodedString) {
```

Comment:
There's an existing `decodeURIComponent` function referenced elsewhere in this file. If they do the same thing, please remove this new function and just use the existing one instead.

Reply:
they don't do the same thing
- decodeURIComponent decodes what has been encoded with rawurlencode in the php side of things
- my decodeEntities function does two things : when adding the value to the textarea element, it will automatically decode any remaining utf8 things (like the single quote, emojis) AND sanitize against any html script (<script>...). Maybe I should name it better like decodeAndSanitize or something

Reply:
Ahhh okay. Yeah please rename it and maybe add a comment as well explaining that it's both decoding utf8 entities and santizing against `<script>`.